src/traces/scattergl/convert.js

```diff
@@ -219,7 +219,7 @@ function convertMarkerSelection(trace, target) {
     if(target.marker && target.marker.symbol) {
         optsOut = convertMarkerStyle(Lib.extendFlat({}, optsIn, target.marker));
     } else if(target.marker) {
-        if(target.marker.size) optsOut.sizes = target.marker.size;
+        if(target.marker.size) optsOut.sizes = target.marker.size / 2;
```

Comment:
Before this commit, `gl2d_point-selection` looked like:

![image](https://user-images.githubusercontent.com/6675409/40630802-bbe30e80-62a3-11e8-8ff2-17dc031bbbcd.png)